### PR TITLE
Use github version of blackfriday/v2 and remove dep files

### PR DESCRIPTION
**Pull request by cmr-informatik**
The blackfriday gopkg.in release is broken when using `go mod vendor`
and building a project dependent on bfchroma with `go build -mod=vendor`
and `go version go1.12.2 gccgo (GCC) 9.2.0 linux/amd64`.

The build fails with:

goroutine 1 [running]:
runtime..z2fdebug.Stack
	/build/gcc/src/gcc/libgo/go/runtime/debug/stack.go:24
runtime..z2fdebug.PrintStack
	/build/gcc/src/gcc/libgo/go/runtime/debug/stack.go:16
main.main
	:0
build derpderp: cannot find module for path github.com/shurcooL/sanitized_anchor_name

Fix this by pointing to the github.com release of blackfriday instead in
bfchroma's go.mod. Take this opportunity to also remove remaining dep
files. Another benefit is that the `replace` directive in go.mod to
workaround some blackfriday release confusion is no longer required
(neither in bfchroma nor in any packages using bfchroma).

Signed-off-by: Christopher M. Riedl <cmr@informatik.wtf>

### CODE REVIEW

**Comment by cmr-informatik** [comment]
I wasn't able to replicate the behavior in https://github.com/Depado/bfchroma/issues/3
I think `blackfriday` has now properly tagged their github release as well. When I tested with my own fork, `go get` appeared to pull the correct `v2.0.1` version:
```
$ go get -v -u github.com/cmr-informatik/bfchroma
...
go: finding github.com/russross/blackfriday/v2 v2.0.1
...
go: downloading github.com/russross/blackfriday/v2 v2.0.1
go: extracting github.com/russross/blackfriday/v2 v2.0.1
...
```

File: example/main.go [hunk 1]
```diff
@@ -3,10 +3,10 @@ package main
 import (
 	"fmt"
 
-	"github.com/Depado/bfchroma"
```
Comment (Depado) [inline example/main.go]: Could you put back the correct import path? :smile:

Comment (Depado) [reply example/main.go]: No problem :D

**Comment by Depado** [comment]
thanks

Thanks a lot for this PR! :smile:

**Pull request merged**